ckanext/iati_generator/plugin.py

```diff
@@ -8,8 +9,12 @@
 
 class IatiGeneratorPlugin(p.SingletonPlugin):
     p.implements(p.IConfigurer)
+    p.implements(p.IBlueprint)
 
     def update_config(self, config_):
         toolkit.add_template_directory(config_, "templates")
         toolkit.add_public_directory(config_, "public")
         toolkit.add_resource("assets", "ckanext-iati-generator")
+
+    def get_blueprint(self):
+        return iati_blueprint
```

Comment:
```suggestion
        return [
            iati_blueprint,
        ]
```

`get_blueprint` must be a list to continue in the future

Reply:
https://github.com/okfn/ckanext-iati-generator/pull/6/commits/108ef9b272d169c9417795f4298fef5e6e579428